src/server/pkg/worker/branch_set.go

```diff
@@ -0,0 +1,245 @@
+package worker
```

Comment:
I am so happy that this file is now called `branch_set.go` instead of `commit_set.go`